doc/maintaining/configuration.rst

```diff
@@ -418,7 +418,7 @@ Example::
 
 Default value: ``None``
 
-Controls if we're caching CKAN's static files, if it's serving them.
+This sets ``Cache-Control`` header for unauthenticated users.
```

Comment:
We should expand this to describe the actual behaviour. IIUC correctly if `ckan.cache_enabled = True`, the header `Cache-Control: public` will be set on requests where the user is not logged in.

Is there a reason why the default value is `None` and not `False`? This seems to be the case [here](https://github.com/ckan/ckan/pull/4781/files#diff-0aa5b9638fa87bb9c7fa9bca15b2c0dbR254)